### PR TITLE
Triboson ZtoLL Vtojj photon

### PR DESCRIPTION
**Pull request by amecca**

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/WLLGTo2L2j_5f_NLO_FXFX/WLLGTo2L2j_5f_NLO_FXFX_madspin_card.dat [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/WLLGTo2L2j_5f_NLO_FXFX/WLLGTo2L2j_5f_NLO_FXFX_madspin_card.dat:13]: the banner could be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZLLGTo2L2j_5f_NLO_FXFX/ZLLGTo2L2j_5f_NLO_FXFX_customizecards.dat [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#put card customizations here (change top and higgs mass for example)
+set param_card mass 6 172.5
+set param_card yukawa 6 172.5
+set param_card mass 25 125.0
+
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZLLGTo2L2j_5f_NLO_FXFX/ZLLGTo2L2j_5f_NLO_FXFX_customizecards.dat:5]: this card may not needed, the higgs mass is 125 in the model, and the top mass will not affect your process here

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZLLGTo2L2j_5f_NLO_FXFX/ZLLGTo2L2j_5f_NLO_FXFX_madspin_card.dat [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+#*    P. Artoisenet, R. Frederix, R. Rietkerk, O. Mattelaer * 
+#*                                                          *
+#*    Part of the MadGraph5_aMC@NLO Framework:              *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZLLGTo2L2j_5f_NLO_FXFX/ZLLGTo2L2j_5f_NLO_FXFX_madspin_card.dat:13]: the banner could be removed

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/WLLGTo2L2j_5f_NLO_FXFX/WLLGTo2L2j_5f_NLO_FXFX_run_card.dat [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+# After showering an MLM-type merging should be applied as well.       *
+# See http://amcatnlo.cern.ch/FxFx_merging.htm for more details.       *
+#***********************************************************************
+ 3        = ickkw            ! 0 no merging, 3 FxFx merging
+#***********************************************************************
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma)                                       *
+#***********************************************************************
+ 30  = bwcutoff
```
Comment (mlizzo) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/WLLGTo2L2j_5f_NLO_FXFX/WLLGTo2L2j_5f_NLO_FXFX_run_card.dat:107]: Hi @amecca , if there is no particular reason to increase the bwcutoff parameter, please set it to 15. It's the recommended value and it should be enough to take Z boson off-shell effects into account.

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZLLGTo2L2j_5f_NLO_FXFX/ZLLGTo2L2j_5f_NLO_FXFX_run_card.dat [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+# After showering an MLM-type merging should be applied as well.       *
+# See http://amcatnlo.cern.ch/FxFx_merging.htm for more details.       *
+#***********************************************************************
+ 3        = ickkw            ! 0 no merging, 3 FxFx merging
+#***********************************************************************
+#
+#***********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma)                                       *
+#***********************************************************************
+ 30  = bwcutoff
```
Comment (mlizzo) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZLLGTo2L2j_5f_NLO_FXFX/ZLLGTo2L2j_5f_NLO_FXFX_run_card.dat:107]: Same comment here

**Pull request merged**